clayui.com/content/blog/2019-10-24-introducing-clay-v3.md

```diff
@@ -0,0 +1,130 @@
+
+All documents related to components in React and CSS, even satellites are all within the same `Components` session. We've put it all together to get content dispersed across the site and help you find documentation more easily.
+
+### React Components
+
+With the release of Clay v3.0.0 we are introducing a rewrite of our components in React, this imposes a major paradigm shift with the framework change from [Metal.js](http://metaljs.com/) to [React.js](https://reactjs.org/).
+
+#### New components
+
+In addition to the existing components we had in v2, we introduced newer components in v3.
+
+| Component | NPM Registry |
+|---|---|
+| [Color Picker](/docs/components/color-picker.html) | `@clayui/color-picker` |
+| [Date Picker](/docs/components/date-picker.html) | `@clayui/date-picker` |
+| [Multi Step Nav](/docs/components/forms/multi-step-nav.html) | `@clayui/multi-step-nav` |
+| [Breadcrumb](/docs/components/navigation/breadcrumbs.html) | `@clayui/breadcrumb` |
+| [Form](/docs/components/forms/form.html), [Input](/docs/components/forms/input.html) | `@clayui/form` |
+| [Vertical Nav, Nav](/docs/components/navigation/vertical-navigation.html) | `@clayui/nav` |
+| [Panel](/docs/components/Panel.html) | `@clayui/panel` |
+| [Popover](/docs/components/popover.html) | `@clayui/popover` |
+| [Tabs](/docs/components/tabs.html) | `@clayui/tabs` |
+| Slider | `@clayui/slider` |
+
+#### Principles
+
+With the framework change that impacts the way we use and think about the use of components, we developed these new components following React's [composition](https://reactjs.org/docs/composition-vs-inheritance.html) approach, you can read more about [Clay's fundamentals of composition in our documentation](/docs/foundations/composing.html) if you want to dig deeper.
+
+### Deprecation warning for Clay v2
+
+Since we are entering v2 and a major framework change for our components, we are starting to deprecated v2 components. This means that we will not be working on new features for v2 but will still continue to support bugs and security issues. We will detail in other posts about how the gradual depreciation of v2 will work.
+
+### Release Process
+
+We are adopting package independent release, releasing packages with their independent version, this will make it easier to make gradual and specific releases of each component. We will test how this release process will work.
+
+#### SemVer
+
+We are adopting the [SemVer](https://semver.org/) versioning system to make it easy for you to upgrade our packages with confidence.
+
+```shell
+MAJOR.MINOR.PATCH
+```
+
+So you can see more often releases of `patch` and consequently` major` but that does not mean there will be big changes like v3 introducing a new rewrite. We hope these releases will be more continuous and help us iterator faster.
```

Comment:
Not super sure what we are trying to communicate here. Can you give a bit more clarification on this part? One thing I might avoid is mention "major" changes since that can be scary to see lol

Reply:
haha yeah, in the end what I meant is that we want to use SemVer more in our favor, versions, beta, alpha ... and send the message that a "major" version is not necessarily a big change, like changing frameworks. I was trying to get the message across that we are not going to do this anytime soon but we are also going to release faster releases, so there might be a lot of "patch" versions for example.

Reply:
Ah gotcha, maybe we could just say something along the lines of..

```suggestion
We hope that by following semver more strictly that it will give developers more confidence in what to expect with each release. Our goal is to provide confidence in new releases by providing as much backwards compatibility as possible and providing sufficient warnings to give users the best experience.
```

This way we don't need to mention any specific version itself and this way we can communicate our overall goal of providing consistency and confidence for users.

Reply:
Looks better I'll add this